seedlet/assets/sass/components/header/_config.scss

```diff
@@ -33,4 +33,7 @@
 	--social-nav--color-link: var(--global--color-foreground);
 	--social-nav--color-link-hover: var(--global--color-primary-hover);
 	--social-nav--padding: calc( 0.5 * var(--primary-nav--padding) );
+
+	--marketing-bar--height: 45px;
+	--wpadmin-bar--height: 46px;
```

Comment:
I'm pretty sure we use this value in the primary nav too so would be good to update there as a follow-up.